constants.js

```diff
@@ -0,0 +1,34 @@
+export const SECTIONS = Object.freeze({
+  WATCH: "WATCH",
+  CHECK: "CHECK",
+  SEARCH: "SEARCH",
+  REGISTER: "REGISTRER",
+  SAFETY: "SAFETY"
+});
+export const FIELD_TYPES = Object.freeze({
+  NAME: "name",
+  DESCRIPTION: "description",
+  PRICE: "price",
+  TAG: "tag"
+});
+export const VALIDATION_STATE = Object.freeze({
+  INITIAL: "INITIAL",
+  SUCCESS: "SUCCESS",
+  FALSE: "FALSE"
+});
+export const ORDER_STATE = Object.freeze({
+  RECENT: "recent",
+  FAVORITEST: "favoritest",
+  NEWEST: "newest"
+});
+export const MODEL_TYPE = Object.freeze({
+  ARTICLE_BEST_LIST: "ARTICLE_BEST_LIST",
+  ARTICLE_STANDARD_LIST: "ARTICLE_STANDARD_LIST",
+  ARTICLE_WITH_COMMENTS: "ARTICLE_WITH_COMMENTS"
+});
+export const EDIT_DELETE_DROPDOWN_LIST = Object.freeze({
+  EDIT_VALUE: "EDIT",
+  EDIT_LABEL: "수정하기",
+  DELETE_VALUE: "DELETE",
+  DELETE_LABEL: "삭제하기"
+});
```

Comment:
@taeyeonkim94 태연님 이것의 이름을 BUTTON_TYPE 으로 하면 좀더 명확한 코드가 될거같은데 어떠신가요?
```suggestion
export const BUTTON_TYPE = Object.freeze({
  edit: {value: 'EDIT', label: '수정하기'},
  delete: {value: 'delete', label:' 수정하기'}
});
```

그러면 이 값을 사용하는 코드에서도 아래처럼 어떤 상수인지 더 직관적으로 알 수 있을거같아요
```js
    if (chosenOption === BUTTON_TYPE.edit.value) setEditMode(true);
    else if (chosenOption === BUTTON_TYPE.delete.value) onDelete(data.id);
```
 대소문자는 취향에 따른거지만, value를 구조분해할당으로 가져와서 쓰느냐 아니면 객체를명시하고 사용할것이냐는 문제가 있잖아요
저는 BUTTON_TYPE을 명시해주는게 좋다고 생각해요. 코드 양이 늘어날수록 `이 상수는 어디서부터 온거지?` 라는 의문이 생길 수 있어요. 항상 그 코드를 보고 있는게 아니니까 본인이 작성했던 코드여도 까먹게되고 나중에 다시봤을때 한 번에 기억이 안날수도 있죠. 그래서 어떤것과 관련된 값인지 코드에 명시를 해주는게 한 눈에 담기 좋을거같아요

Reply:
이해했습니다 근데 BUTTON_TYPE보다는 드랍다운에서 왔으니 EDIT_DELETE_를 뺴고 DROPDOWN_LIST로 사용하는건 별로일까요?

Reply:
수정했습니다